shared/components/input/Input.jsx

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Input = styled.input`
+  background-image: #fff;
+  border: solid 1px #ced0da;
+  border-color: ${props => props.isValid === false && '#fb6375'};
```

Comment:
What value will `border-color` have if it evaluates to `true`? Also, the `isValid` prop should be a boolean anyway.

Reply:
I use 3 different states to regulate the prop isValid. true, false and undefined.